src/components/dev-hub/series.js

```diff
@@ -31,99 +31,112 @@ const BORDER_GRADIENT = `linear-gradient(
 );`;
 
 const activeLiStyles = css`
-    font-size: 22px;
+    font-size: ${fontSize.xsmall};
+    padding-left: 1px;
+    line-height: ${lineHeight.small};
     &:after {
         /* Adjust the background bullet position for the bull's eye text */
-        font-size: 54px;
-        top: 1px;
+        top: -1px;
     }
 `;
 
 const activeLinkStyles = css`
     font-weight: bold;
 `;
+const pastLinkStyles = css`
+    color: ${PAST_LINK_COLOR};
+    &:visited {
+        color: ${PAST_LINK_COLOR};
+    }
+    &:hover {
+        color: ${colorMap.darkGreen};
+    }
+`;
 
 const Breadcrumb = styled('li')`
     display: flex;
-    padding-bottom: ${size.small};
+    padding-bottom: ${size.mediumLarge};
     position: relative;
     z-index: ${layer.back};
+    :last-of-type {
+        padding-bottom: 0;
+        > div {
+            &:before {
```

Comment:
This addresses the `ul` border extending too far on a wrapped last element. The `bottom` position of the border was always fixed.